payload-schemas/schemas/common/installation-full.schema.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Anyone have suggestions for a better file name? I'm not too keen on it as it is right now

Reply:
* `installation-complete` ?
* `installation-detailed` ?

Reply:
What's the difference(s) between this and `installation`?

Generally I'd recommend using the difference in the name, i.e `InstallationWith<Property>`.
Depending on what the differences are, it could be better to rename this to `Installation` and rename the existing schema to something else.

Reply:
This one is for the `installation` and `installation_repositories` events, the `installation` schema is for every event whenever it's sent by an app

Reply:
> the installation schema is for every event whenever it's sent by an app

Is that a slimmed down version of the "full" schema? if so, I'd recommend renaming it to `installation-lite` or similar - I think we should be favoring matching the models on the backend for these sorts of schemas first, since we're going to have a number of "lite" versions in events just due to the nature of REST APIs.

Reply:
It only has one of the properties from the other `installation` schema, the `id` property, plus an additional property `node_id`

Reply:
ah yeah - then 💯 this is what I expected.

I'd call this `Installation` and the current `installation` common schema should be something like `InstallationRef`.

(Actually, that's exactly what it should be and what I'm meaning - the underlying model on the backend is `installation`, and elsewhere we have references to that model 🎉. It's what I did for `repo` too)